editor/src/plugins/wrapped-tpc-templates-button.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { createPortal, useLayoutEffect, useRef } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import TpcTemplatesButton from '../components/tpc-templates-button';
+
+/**
+ * A wrapper for the TPC templates button that renders it inside post header toolbar.
+ */
+function WrappedTpcTemplatesButton() {
```

Comment:
NIT: We can keep consistent and use arrow functions. And if there's only one export in the function, export default so you don't have to worry about the component naming during the import as I can see in case of this component.